src/modules/form/util/formUtil.ts

```diff
@@ -453,6 +454,22 @@ export const shouldEnableItem = ({
   }
 };
 
+const isPicklistOption = (
```

Comment:
We already have a type guard function called isPickListOption -- did you miss that, or did you need a separate one for the fragment?

Reply:
ah I missed that, thanks